docs/developer-guide/emt-architecture-overview.md

```diff
@@ -386,6 +386,36 @@ PERSISTENT_BIND_PATHS="
   additional bind mounts such as `/etc/rancher`, `/etc/cni`, `/etc/kubernetes`,
   `/var/lib/rancher`.
 
+## Boot Optimization
```

Comment:
@sgolebiewski-intel have this been validated on EMT?

Reply:
There was a developer level checkout with SBL. Not sure if a formal validation was done.